nginx/cernopendata.conf

```diff
@@ -1,7 +1,12 @@
 
+proxy_cache_path /tmp/nginx-cache-cod levels=1:2 keys_zone=cache_cod:100m inactive=60m max_size=500m use_temp_path=off;
```

Comment:
Investigate using `/var/cache/nginx` which is emptyDir on OpenShift by default already.